spec/requests/api/v1/admin/dimensions_spec.rb

```diff
@@ -0,0 +1,35 @@
+# frozen_string_literal: true
+
+require 'rails_helper'
+
+describe 'Admin Dimensions' do
+  let(:user)    { Fabricate(:user, role: UserRole.find_by(name: 'Admin')) }
+  let(:token)   { Fabricate(:accessible_access_token, resource_owner_id: user.id, scopes: scopes) }
+  let(:headers) { { 'Authorization' => "Bearer #{token.token}" } }
+  let(:account) { Fabricate(:account) }
+
+  describe 'GET /api/v1/admin/dimensions' do
+    context 'when not authorized' do
+      it 'returns http not found' do
+        get api_v1_admin_dimensions_path, params: { account_id: account.id, limit: 2 }
```

Comment:
I think it makes sense hardcoding the actual paths in request specs, as far as API tests are concerned, as a change in paths would constitute a breaking API change

Reply:
Updated to use path strings